### PR TITLE
UI/userlist

### PR DESCRIPTION
**Pull request by nontan0518**
# Conflicts:
#	.gitignore
#	public/mix-manifest.json
#	resources/views/layouts/app.blade.php
#	webpack.mix.js

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,9 @@
 /node_modules
 /public/hot
 /public/storage
+/public/css
```
Comment (keanbillyfernandez) [inline .gitignore:4]: The file .gitignore is okay not to push

Comment (nontan0518) [reply .gitignore:4]: The file .gitignore is okay not to push　？　so I just leave them? I don't know what your meaning

Comment (keanbillyfernandez) [reply .gitignore:4]: @nontan0518 , The .gitignore is okay not to push. So just leave them in your local and then just add files if needed. But no need to push this file going to main branch

Comment (nontan0518) [reply .gitignore:4]: thank you so much so just leave them

File: DEVSENSE/packages-cache/asm89/stack-cors.v2.1.1.json.gz
Comment (Ditzler01) [inline DEVSENSE/packages-cache/asm89/stack-cors.v2.1.1.json.gz:1]: Please delete these files under DEVSENSE directory.

**Pull request merged**